toc.md

```diff
@@ -610,28 +756,50 @@ module Workerholic
 end
 ```
 
+Let's walk through this algorithm in the context of our example.
+- the average job count per worker is: `52,840 / 21 = 2,516.19`
+- we provision `Queue 1` with: `50,000 / 2,516.19 ~ 20`
+- we now have 24 workers assigned and 1 left to assign
+- then, we provision `Queue 2` with: `2,000 / 2,516.19 ~ 1`
+- we have now provisioned the queues with all 25 workers
+
 ![ASP_diagram_2](/images/ASP_diagram_2.png){:width="350"}
 
-Once an average is calculated, Workerholic divides each of its queue sizes by the average, get a workers count, and assign that many extra workers to their respective queues.
+Next, we will start looking at how, as BJP builders, we can help a user make better decision about his/her background job system.
 
 ### Reporting
-Let's move on to reports. It is an important feature because it allows a developer to gain insight into the state of overall jobs, the job types, jobs that failed, and how many jobs are completed over time, which the developer can use to make optimized decisions.
 
-![reporting_web_ui](/images/reporting_web_ui.png)
+Reporting is an important feature to implement when building a BJP. It allows developers to gain insight into the state of overall jobs, the job types, jobs that failed, and how many jobs are completed over time, which the developer can use to make optimized decisions about their background job system.
```

Comment:
into the overall state of jobs

which the developer can use to make better decisions about adjustments needed to the background job system.